_docs/v2.6/configure/scanner/descriptions.md

```diff
@@ -145,6 +145,30 @@ For examples of how to define scanner rules for network interfaces, see the
 [`instance_network_interface_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/instance_network_interface_rules.yaml)
 rule file.
 
+## Lien scanner
+Allow customers to ensure Liens for their projects exist and are configured correctly.
+
+For examples of how to define scanner rules for lien, see the
+[`lien_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/lien_rules.yaml)
+rule file.
+
+## Location scanner
+Allow customers to ensure their resources are located only in the intended 
+locations, or are not located in specific regions at all. Set guards around 
+locations as part of automated project deployment.
+
+For examples of how to define scanner rules for location, see the
+[`location_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/location_rules.yaml)
+rule file.
+
+## Log sink scanner
```

Comment:
qq: is this only from 2.6?  Maybe my memory is off, but seems we had this from a while back?

Reply:
Added to 2.5 2.4 and 2.3